ubpf_plugin/ubpf_plugin.cc

```diff
@@ -133,6 +134,12 @@ int main(int argc, char **argv)
         jit = false;
         args.erase(args.begin());
     }
+#if defined(UBPF_HAS_ELF_H)
+    if (args.size() > 0 && args[0] == "--elf") {
```

Comment:
This is not something that you need to act on, but ... do we want to always enable this option and print a message saying that it was specified and not available? That might give better feedback to the user rather than simply saying, "Parameter not recognized" (or whatever it says).